src/main/java/webs/hillel/StudentDAO.java

```diff
@@ -0,0 +1,78 @@
+package webs.hillel;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class StudentDAO {
+    private static final Logger logger = Logger.getLogger(StudentDAO.class.getName());
```

Comment:
Please follow the naming conventions - if you create a constant (static final field of any type) then you should name it upper case, like this: `logger` -> `LOGGER`